docs/azure/sdk/snippets/aspnetcore-guidance/MinApiSample/Program.cs

```diff
@@ -0,0 +1,80 @@
+using Azure.Identity;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Microsoft.Extensions.Azure;
+using Azure.Messaging.ServiceBus;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddOpenApi();
+
+builder.Services.AddAzureClients(clientBuilder =>
+{
+    // Register clients using a config file section
+    clientBuilder.AddSecretClient(
+        builder.Configuration.GetSection("KeyVault"));
+
+    clientBuilder.AddBlobServiceClient(
+        builder.Configuration.GetSection("Storage"));
+
+    // Register clients using a specific config key-value pair
+    clientBuilder.AddServiceBusClientWithNamespace(
+        builder.Configuration["ServiceBus:Namespace"]);
+
+    // Register a subclient for each Azure Service Bus Queue
+    var queueNames = new string[] { "queue1", "queue2" };
```

Comment:
Use a C# 12 collection expression to simplify this line a bit:
```suggestion
    string[] queueNames = [ "queue1", "queue2" ];
```